tests/test_archiver.py

```diff
@@ -239,14 +240,24 @@ def test_archived_file(self, url):
         result = json.loads(update_resource(self.config, res_id))
 
         assert result['cache_filepath']
-        assert os.path.exists(result['cache_filepath'])
 
-        with open(result['cache_filepath']) as f:
-            content = f.readlines()
-            assert len(content) == 1
-            assert content[0] == "test"
+        from ckan.lib.uploader import ResourceUpload as DefaultResourceUpload
+        if not hasattr(DefaultResourceUpload, "download"):
```

Comment:
"if not...else" is a double negative.

Reply:
wanted to leave the old code first so not check.